server/lib/emailer.ts

```diff
@@ -15,6 +15,50 @@ import { MAbuseFull, MAbuseMessage, MAccountDefault, MActorFollowActors, MActorF
 import { MCommentOwnerVideo, MVideo, MVideoAccountLight } from '../types/models/video'
 import { JobQueue } from './job-queue'
 
+const sanitizeHtml = require('sanitize-html')
+const markdownItEmoji = require('markdown-it-emoji/light')
+const MarkdownItClass = require('markdown-it')
+const markdownIt = new MarkdownItClass('default', { linkify: true, breaks: true, html: true })
+
+markdownIt.enable([
```

Comment:
I forgot `autolink` but maybe because `linkify` and `newline` are already defined in parser options as `breaks: true` and `linkify: true` line 21 ?

Reply:
I don't know how this interacts within `markdown-it` honestly. If you feel it shouldn't be here, then it's fine. Otherwise I'd stick with staying consistent with the client.